calcite/src/main/java/husky/sql/calcite/table/SimpleTestTable.java

```diff
@@ -5,27 +5,77 @@
 import org.apache.calcite.linq4j.Linq4j;
 import org.apache.calcite.rel.type.RelDataType;
 import org.apache.calcite.rel.type.RelDataTypeFactory;
+import org.apache.calcite.rel.type.RelDataTypeFactory.Builder;
 import org.apache.calcite.rel.type.RelProtoDataType;
 import org.apache.calcite.schema.ScannableTable;
 import org.apache.calcite.schema.impl.AbstractTable;
 import org.apache.calcite.sql.type.SqlTypeName;
 
+import java.io.FileNotFoundException;
```

Comment:
How about write a new table class and a new model.json instead of overwriting the existing one?

Reply:
> How about write a new table class and a new model.json instead of overwriting the existing one?

Sure.